app/src/views/features/sessions/SessionsIndexPageContainer/SessionsIndexPage/OnboardingView.tsx

```diff
@@ -249,6 +163,12 @@ const SessionOnboardingView: React.FC<SessionOnboardProps> = ({
     redirectToSettingsPage();
   }, [redirectToSettingsPage]);
 
+  useEffect(() => {
+    if (user.loggedIn && refParam === "producthunt" && !isExtensionInstalled()) {
```

Comment:
We don't need refParam === 'producthunt' check here. It should open even if ref is not producthunt

Reply:
This is not needed as install extension cta gets shown when user clicks on `Start Recordin`